flexmeasures_client.py

```diff
@@ -58,23 +58,24 @@ def get_device_message(self, kwargs, **fnc_kwargs):
 
         Pass the UDI event id using kwargs["udi_event_id"]=<udi_event_id>.
         """
-        url = self.args["fm_api"] + "/" + self.args["fm_api_version"] + "/getDeviceMessage"
         udi_event_id = kwargs["udi_event_id"]
+        url = self.args["fm_api"] + "/" + self.args["fm_api_version"] + "/sensors/" + str(self.args["fm_quasar_sensor_id"]) + "/schedules/" + udi_event_id
         message = {
-            "type": "GetDeviceMessageRequest",
-            "event": self.args["fm_quasar_entity_address"] + ":" + str(udi_event_id) + ":soc",
+            # "type": "GetDeviceMessageRequest",
+            # "event": self.args["fm_quasar_entity_address"] + ":" + str(udi_event_id) + ":soc",
             "duration": self.args["fm_schedule_duration"],
         }
         res = requests.get(
             url,
             params=message,
             headers={"Authorization": self.fm_token},
         )
-        if res.status_code != 200:
-            self.log(f"GetDeviceMessage failed with response {res.json()}")
-            self.handle_response_errors(message, res, "GET device message", self.get_device_message, kwargs,
-                                        **fnc_kwargs)
-            return
+        self.log(f"Result code: {res.status_code}")
+        # if res.status_code != 200:
+        #     self.log(f"GetDeviceMessage failed with response {res.json()}")
+        #     self.handle_response_errors(message, res, "GET device message", self.get_device_message, kwargs,
+        #                                 **fnc_kwargs)
+        #     return
```

Comment:
We should check whether we still need this in case of an expired auth token.

Reply:
Could use some help...